core/common/src/main/java/alluxio/Configuration.java

```diff
@@ -200,11 +209,11 @@ public static void set(String key, String value) {
    * @return the value for the given key
    */
   public static String get(String key) {
-    if (!PROPERTIES.containsKey(key)) {
+    if (!PROPERTIES_MAP.containsKey(key)) {
```

Comment:
For many of these getters the typical flow is "check contains", then run `get`. However, why don't we just run `get` directly, and deal with the `null` return if it didn't exist?

Maybe @apc999 has thoughts?